app/Controllers/GeneriController.php

```
@@ -84,6 +84,100 @@ public function store(Request $request, Response $response, \mysqli $db): Respon
+            }
+
+            $repo->update($id, $updateData);
+            $_SESSION['success_message'] = __('Genere aggiornato con successo!');
+        } catch (\Throwable $e) {
+            $_SESSION['error_message'] = __('Errore nell\'aggiornamento del genere: ') . $e->getMessage();
+        }
+
+        return $response->withHeader('Location', "/admin/generi/{$id}")->withStatus(302);
+    }
+
+    public function destroy(Request $request, Response $response, \mysqli $db, int $id): Response
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)